Source/Core/DolphinQt2/Translate.h

```diff
@@ -0,0 +1,270 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+#include <QTranslator>
+
+class DQTranslator : public QTranslator
```

Comment:
This can just be called Translator. I haven't been prefixing DQt stuff with DQ, so lets be consistent.